tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -46,12 +46,18 @@ sub run {
     wait_screen_change { send_key "alt-o"; };
     # Check that entry was correctly edited in /etc/hosts
     select_console "root-console";
-    assert_script_run q#grep '195\.135\.221\.134\sdownload\.opensuse\.org\sdownload-srv' /etc/hosts#;
+    assert_script_run q#grep '195\.135\.221\.134\s*download\.opensuse\.org\s*download-srv' /etc/hosts#;
 }
 
 # Test ends in root-console, default post_run_hook does not work here
 sub post_run_hook {
     set_var('YAST2_GUI_TERMINATE_PREVIOUS_INSTANCES', 1);
 }
 
+sub post_fail_hook {
+    my ($self) = shift;
+    $self->SUPER::post_fail_hook;
+    upload_logs('/etc/hosts');
```

Comment:
I would swap this two lines to get first the specific and later the generic, but perhaps it is a matter or taste as I'm not sure if we have precedents of SUPER::post_fail_hook failing.

Reply:
Yes, you're right. I've swapped them. Thanks.